src/native/libs/System.Globalization.Native/static-build/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(System.Globalization.Native.User.Machine.Build C)
```

Comment:
The PR seems to be using multiple different names for the same thing:

- local build (PR title)
- static build (directory name)
- user machine build (this project name)

It would be nice to standardize on a single name. I like "local build" the best out of these options.